opennebula/helpers.go

```diff
@@ -143,3 +144,15 @@ func mergeSchemas(schema map[string]*schema.Schema, schemas ...map[string]*schem
 
 	return schema
 }
+
+func ParseInt64FromInterface(i interface{}) (int64, error) {
+	switch v := i.(type) {
+	case float64:
+		return int64(v), nil
+	case string:
+		if r, err := strconv.ParseInt(v, 10, 0); err == nil {
```

Comment:
strconv.ParseInt(v, 10, 0) uses bitsize 0 and will return Int. For old 32-bit platforms it will be Int32 and not Int64

Reply:
So the return type is `i int64` anyway, then provider is full of constructions like `strconv.ParseInt(asd, 10, 0)`, imo this should be reviewed and corrected as a dedicated effort. :thinking: